README.md

```diff
@@ -91,6 +83,32 @@ Host Docker storage drivers of `overlay` and `aufs` are supported automatically.
     vagrant ssh
     ```
 
+1. Configure DC/OS Docker:
+
+    Generate `make-config.mk` automatically, interactively, or manually.
+
+    **Automatic Mode:**
+
+    ```console
+    ./configure --auto
+    ```
+
+    **Interactive Mode:**
+
+    ```console
+    ./configure
+    ```
+
+    **Manual Mode (example):**
+
+    ```console
+    cat > make-config.mk << EOM
+    MASTERS := 3
+    EOM
+    ```
+
+    See [make-defaults.mk](make-defaults.mk) for a full list of manually configurable options.
```

Comment:
This is tricky but this isn't the full list of manually configurable options - in fact I regularly use `MASTER_MOUNTS` and the only reason for that existing is for configuration - it is empty by default.

However, every variable with `:=` is manually configurable.